pandoc.cabal

```diff
@@ -547,6 +548,7 @@ library
   hs-source-dirs:  src
 
   exposed-modules: Text.Pandoc,
+                   Text.Pandoc.Readers.Roff,
```

Comment:
exposed this for my `lexroff.hs` thing which I didn't commit, move back to `other-modules` before merge